lib/v2/cnblogs/cate.js

```diff
@@ -0,0 +1,30 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type;
+
+    const link = `https://www.cnblogs.com/cate/${type}`;
+    const response = await got(link);
+    const data = response.data;
+
+    const $ = cheerio.load(data);
+    const list = $('#post_list article')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.find('.post-item-title').text(),
+                link: item.find('.post-item-title').attr('href'),
+                pubDate: parseDate(item.find('.post-item-foot .post-meta-item span').text(), 'YYYY-MM-DD HH:ii:ss'),
```

Comment:
For me, the time shown are in `YYYY-MM-DD HH:mm` only
```suggestion
                pubDate: parseDate(item.find('.post-item-foot .post-meta-item span').text(), 'YYYY-MM-DD HH:mm'),
```
the format is listed here: https://day.js.org/docs/en/parse/string-format

You may want to adjust the timezone as well
docs: https://docs.rsshub.app/joinus/pub-date.html